PlaybackLab/MultiplePlayersScrollingViewSampleApp/MultiplePlayersScrollingViewSampleApp/Utils/OptionDataSource.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  OptionDataSource.swift
+//  MultiplePlayersScrollingViewSampleApp
+//
+//  Copyright © 2019 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class OptionDataSource: NSObject {
```

Comment:
See how it's done in `DTO Swift` sample app

Reply:
Done